app/code/Magento/Catalog/Block/Product/ImageBuilder.php

```diff
@@ -7,7 +7,7 @@
 
 namespace Magento\Catalog\Block\Product;
 
-use Magento\Catalog\Helper\ImageFactory as HelperFactory;
+use Magento\Catalog\Block\Product\ImageFactory as HelperFactory;
```

Comment:
We have to change alias, previously it was helper factory, now it becomes to image block factory

Reply:
So I have to replace HelperFactory with ImageBlockFactory right?